pootle/core/views.py

```diff
@@ -464,7 +465,8 @@ def is_admin(self):
 
     @property
     def language(self):
-        return None
+        if self.tp:
+            return self.tp.language
```

Comment:
@phlax, I wonder why didn't you put this to `UnitEditJSON` class. What have I missed?